apis/js-apis/search.md

```diff
@@ -24,6 +24,15 @@ The api.search function also accepts the following options:
 
 The function automatically loads session parameters required for personalization & segments in the background.
 
+In order to request custom fields, add the entries `"customFields.key", "customFields.value"` to the requested product fields.
```

Comment:
I would change this to:
In order to request custom fields, add the entries `"customFields.key"` and `"customFields.value"` to the requested product fields.